api/buildings/models.py

```diff
@@ -248,3 +257,72 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class ApprovedImage(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(status=ImageFile.ACCEPTED)
+
+
+class ImageFile(models.Model):
+    PENDING = 0
+    ACCEPTED = 1
+    REJECTED = -1
+
+    IMAGE_STATUS_CHOICES = [
+        (PENDING, _("Pending")),
+        (ACCEPTED, _("Accepted")),
+        (REJECTED, _("Rejected")),
+    ]
+
+    def set_images_limit(building):
```

Comment:
I think that a name like `check_image_limit` would be more clear about this function's purpose.

Reply:
Good suggestion, I've updated the name.